packages/astro/src/internal/fetch-content.ts

```diff
@@ -0,0 +1,22 @@
+export async function fetchContent(importMetaGlobResult: Record<string, () => Promise<any>>, url: string) {
+    if (typeof importMetaGlobResult === 'string') {
+      throw new Error(`[deprecated] "Astro.fetchContent(str)" is now "await Astro.fetchContent(import[dot]meta[dot]glob(str))"`);
```

Comment:
Needed because our import.meta.glob find-replace gets it otherwise. 

(fwiw I think Vite's is smarter than Snowpack's and uses the full AST)

Reply:
Could have sworn we were checking for `import.meta.glob` inside comments. Would definitely like to fix this.